### DISCUSSION TITLE
Treesitter executable no found (windows)

### DISCUSSION THREAD

**Discussion by ANP-Oxy** [nvim-treesitter/nvim-treesitter#6511] [Q&A] [answered]
I'm trying to setup neovim on windows 11

neovim version - NVIM v0.9.5

I installed treesitter with lazy.nvim

when I run checkhealth it shows following 

**vim-treesitter: require("nvim-treesitter.health").check()

Installation ~
- WARNING `tree-sitter` executable not found (parser generator, only needed for :TSInstallFromGrammar, not required for :TSInstall)
- OK `node` found v20.12.2 (only needed for :TSInstallFromGrammar)
- OK `git` executable found.
- OK `zig` executable found. Selected from { vim.NIL, "cc", "gcc", "clang", "cl", "zig" }
- OK Neovim was compiled with tree-sitter runtime ABI version 14 (required >=13). Parsers must be compatible with runtime ABI.

OS Info:
{
  machine = "x86_64",
  release = "10.0.22631",
  sysname = "Windows_NT",
  version = "Windows 11 Home Single Language"
} ~ **

I tried to install parser for python by using **:TSInstall python** but that doesn't work. It doesn't throw any error either.

**Comment by clason** [comment]
Treesitter means the CLI, which you need to install separately. But you don't need that for installing Python. If there's no output you can share, there's nothing to go on. Please try creating a minimal config we can test and then open a proper issue.

**Reply by ANP-Oxy** [reply, marked as answer]
my bad for making a clear post I'm still a newbie. Anyways the issue was with the c compiler that was being used. Zig wasn't really working so I tried clang and TS wasn't recognizing it even though it was in path. Finally the issue was solved by installing mingw and setting it explicitly as compiler in treesitter config.
